run_Wochenende.py

```diff
@@ -282,7 +301,93 @@ def runAfterQC(stage_infile):
     return stage_outfile
 
 
-def runFastpSE(stage_infile, noThreads):
+
+def runTrimGaloreSE(stage_infile, noThreads, nextera):
+    # use for Nextera - single end reads
+    stage = "TrimGalore - SE"
+    print("######  " + stage + "  ######")
+    prefix = stage_infile.replace(".fastq", "")
+    stage_outfile = prefix + ".trm.fastq"
+    trimNextera=""
+    if nextera:
+        trimNextera = "--nextera"
+    trim_galore_cmd = [
+	#trim_galore --nextera --dont_gzip --cores 12 --2colour 20  x_R1.fastq > out_R1.fastq
+        path_trim_galore,
+        trimNextera,
+        "--dont_gzip ",
+        "--length 36 ",
+        "--suppress_warn ",
+#        "--adapter_fasta=" + adapter_path,
+        "--cores " + noThreads,
+        "--2colour " + trim_galore_min_quality,
+        stage_infile,
+        #" > ",
+        #stage_outfile,
+    ]
+    rename_cmd = [
+        #trim_galore creates prefix_trimmed.fq . Rename this to outfile
+        "mv" ,
+        prefix + "_trimmed.fq",
+        stage_outfile,
+    ]
+    #runStage(stage, trim_galore_cmd)
+    trimGaloreCmdStr = " ".join(trim_galore_cmd)
+    renameStr = " ".join(rename_cmd)
+
+    try:
+        # could not get subprocess.run, .call etc to work with pipes and redirect '>'
+        print("trimGaloreCmdStr: " + trimGaloreCmdStr)
+        print("renameStr: " + renameStr)
+        os.system(trimGaloreCmdStr)
+        os.system(renameStr)
+
+
+    except:
+        print("Error running trim_galore")
+        sys.exit(1)
+
+    rejigFiles(stage, stage_infile, stage_outfile)
+    return stage_outfile
+
```

Comment:
looks fine, tested the pipeline and nextera transposase sequences are removed from 3' end.